partner_firstname/README.rst

```diff
@@ -0,0 +1,86 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: https://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+================================
+Partner first name and last name
+================================
+
+
+Configuration
+=============
+
+You can configure some common name patterns for the inverse function
+in Settings > General settings:
+
+* Lastname Firstname: For example 'Anderson Robert'
+* Lastname, Firstname: For example 'Anderson, Robert'
+* Firstname Lastname: For example 'Robert Anderson'
+
+After applying the changes, you can recalculate all partners name clicking
+"Recalculate names" button. Note: This process could take so much time depending
+how many partners there are in database.
+
+You can use *_get_inverse_name* method to get lastname and firstname from a simple string
+and also *_get_computed_name* to get a name form the lastname and firstname.
+These methods can be overridden to change the format specified above.
+
+
+Usage
+=====
+
+
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/134/11.0
+
+For further information, please visit:
+
+* https://www.odoo.com/forum/help-1
+
+
+Known issues / Roadmap
```

Comment:
@gdgellatly IMO if not use then Remove it

Reply:
@nikul-serpentcs This whole file gets auto replaced, no need to look at it

Reply:
@gdgellatly if you want to generate it by yourself, you can use the `oca-gen-addon-readme` script from https://github.com/OCA/maintainer-tools#installation